src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -65,6 +65,44 @@ public sealed class CSharpCompilationOptions : CompilationOptions, IEquatable<CS
         {
         }
 
+        internal CSharpCompilationOptions(
```

Comment:
Usages of the constructor that use the features parameter (i.e. CommandLineParser) don't provide all the parameters of the constructor, letting some be their default parameters. The one with the features parameter doesn't have optional parameters, so I'd have to extract the defaults into the CommandLineParser to be able to use that constructor.